src/main/java/uk/gov/hmcts/reform/sscscorbackend/controllers/CitizenController.java

```diff
@@ -61,10 +61,6 @@ public CitizenController(CitizenLoginService citizenLoginService,
             @ApiParam(value = "user authorisation header", example = "Bearer eyJhbGciOiJIUzUxMiJ9.eyJzdW") @RequestHeader(AUTHORIZATION) String authorisation,
             @ApiParam(value = "tya number for an user and appeal", example = "A123-B123-c123-Dgdg") @PathVariable("tya") String tya) {
 
-        if (!idamService.verifyTokenSignature(authorisation)) {
```

Comment:
It's ok to remove these lines?

Reply:
Yes now we have spring-security which authorised based on token